src/index.ts

```diff
@@ -1,19 +1,80 @@
 import { Probot } from "probot";
 import fs from 'fs';
+import { V1Secret } from '@kubernetes/client-node/dist/gen/model/v1Secret';
 const k8s = require('@kubernetes/client-node');
+const { createAppAuth } = require("@octokit/auth-app");
+require('dotenv').config();
 
+
+const EXPIRATION_THRESHOLD = 5 * 60000;
+
+// K8s client for using k8s apis.
 const kc = new k8s.KubeConfig();
 kc.loadFromDefault();
 const k8sApi = kc.makeApiClient(k8s.CoreV1Api);
 const k8sContext = kc.getCurrentContext();
-const namespace = (() => {
+const k8snamespace = (() => {
   if (k8sContext === 'inClusterContext') {
     return fs.readFileSync('/var/run/secrets/kubernetes.io/serviceaccount/namespace', 'utf8');
   } else {
     return k8sContext.split('/')[0];
   }
 })();
-console.log(`Operating in the namespace: ${namespace}`);
+console.info(`Operating in the namespace: ${k8snamespace}`);
+
+/**
+* Function that get the new installation token
+* @param    {Number} installId    Installation ID
+* @return   {Array} [Token, ExpiryAt] List containing installation token and expiry time
+*/
+async function getToken(installId: Number) {
```

Comment:
do we expect to use this code anywhere else than in the `checkExpiryUpdateToken` func? It's 2 lines of code, why don't we inline it directly into `checkExpiryUpdateToken`?

Reply:
i thought of levaing it, thinking maybe something can come up in future.
will merge it